scripts/release.py

```diff
@@ -25,6 +27,20 @@
 
 RELEASE_BRANCH_PREFIX = "prepare-release-"
 
+PRERELEASE_VERSION_PATTERN = re.compile(r"^(a|rc)([1-9]\d*)$")
+
+RELEASE_BRANCH_PATTERN = re.compile(r"^\d+\.\d+\.x$")
+
+
+class Version(BaseVersion):
+    """
+    A PEP440 compatible version that supports prereleases:
+    https://www.python.org/dev/peps/pep-0440/#pre-releases
+    """
+
+    def __str__(self):
+        return super().__str__().replace("-", "")
```

Comment:
what's this for? 😅

Reply:
that's what I said about strict semver: PEP440 diverges from it. `semantic_version` package implements strict semver, making the format for prerelease like `1.10.0-a1` or `1.10.0-rc1`. It's definitely a hack :/

Reply:
ah makes sense - can you add that as a comment?